pages/common/mediainfo.md

```diff
@@ -0,0 +1,15 @@
+# mediainfo
+
+> Display metadata from video and audio files.
+
+- Display metadata for a given file in the console:
+
+`mediainfo {{file}}`
+
+- Store the output to a given file apart from displaying in the console:
```

Comment:
"apart from displaying in the console" feels a bit confusing. Does it display the data into the console too? Or not?

Also, the `--Logfile` bit has a capital letter. Was this intentional, or does it not work without it?

Reply:
Ah yes, I should change it to "along with displaying in the console".

> Also, the --Logfile bit has a capital letter.

Hmm, it does work with `--logfile`. But `--Logfile` is what it says in its help section. So I would rather be safe.

Reply:
Ah, ok. Let's stick with the safe option then.